vector/src/main/res/values-ar/strings.xml

```diff
@@ -71,14 +71,6 @@
     <string name="rooms_directory_header">دليل الغرف</string>
     <string name="no_room_placeholder">لا غرف</string>
     <string name="no_public_room_placeholder">لا غرف عمومية متوفرة</string>
-    <plurals name="public_room_nb_users">
```

Comment:
I think this resource has been deleted by mistake, no? It's not deleted in the main `strings.xml` file. This remark is applicable to all other translations as well